cli/config/config.go

```diff
@@ -102,11 +101,17 @@ func GetSeedImage() string {
 
 func GetPackageName() string {
 	metadata := GetMetaData()
+	prefix := PackagePrefix
+	suffix := "tar.zst"
+
+	if IsZarfInitConfig() {
+		prefix = "zarf"
+	}
+
 	if metadata.Uncompressed {
-		return PackagePrefix + metadata.Name + ".tar"
-	} else {
-		return PackagePrefix + metadata.Name + ".tar.zst"
+		suffix = "tar"
 	}
+	return fmt.Sprintf("%s-%s-%s.%s", prefix, strings.ToLower(metadata.Name), GetArch(), suffix)
```

Comment:
Am I wrong in thinking that some packages will have images that support multiple architectures (or be so simple that they don't actually have an images in the package)?

It doesn't seem like including the current arch into the package name make sense in those situations.

Reply:
yeah that's an interesting point, but zarf checks arch regardless because it is needed for images and crane (with how we use it) doesn't support multiarch builds rn.  I think it's a worth UX to mixing naming conventions or leave off the arch when it is really important most of the time.  This will especially be true with the mutating webhook work since it will assume all k8s resources use the internal registry by default.